src/utils/wallet.js

```diff
@@ -97,7 +97,8 @@ const wallet = {
    * @return {boolean} boolean indicating if address is valid
    */
   validateAddress(address) {
-    const networkName = LOCAL_STORE.getNetwork() || 'mainnet';
+    const networkSettings = LOCAL_STORE.getNetworkSettings();
+    const networkName = networkSettings ? networkSettings.network : 'mainnet';
```

Comment:
thought(non-blocking): This could also be benefited by this "safeGet" suggestion.